packages/bridge/bridge-react/src/lifecycle.ts

```diff
@@ -0,0 +1,29 @@
+import { getInstance } from '@module-federation/runtime';
+import helper from '@module-federation/runtime/helpers';
```

Comment:
i wonder if we should make this use named exports in the future, so that better tree shake or destructure is possible later. (for the helpers stuff)
`import {global} from helper`